_i18n/es/resources/user-guides/restore_account.md

```diff
@@ -1,53 +1,53 @@
 
--  You will then be prompted with "Restore from specific blockchain height (optional, default 0):" Default will start the restore process from the beginning of the Monero blockchain. If you don't know the specific blockchain height, just hit enter. (Specifying a specific blockchain height will start the restore process from that specific height. This will save a bit of time in scanning, if you know what starting blockchain height your initial funds were transacted for this specific account.)
+- Se pedirá ahora el restaurar desde una altura específica en la blockchain (Restore from specific blockchain height), opcional, default 0: Default empezará el proceso de restauración desde el inicio de la blockchain de Monero. Si no conoces la altura de bloque específica, sólo presiona Enter. (Dar una altura de bloque específica iniciará el proceso de restauración desde esa misma altura. Esto ahorrará un poco de tiempo en el escaneo, si conoces la altura de bloque en la cual tus fondos iniciales fueron tramitados para tu cuenta en específico).
 
-After you have entered the 25 word mnemonic seed and have chosen your specific blockchain height, monero-wallet-cli will generate the same public address and view key as your old wallet and begin the refresh process automatically. (Please be patient as the refresh process may take a while.)
+Después de haber ingresado la semilla mnemónica de 25 palabras y haber elegido la altura de bloque específica, monero-wallet-cli generará la misma llave de visualización y dirección pública que tu viejo monedero e iniciará el proceso de actualización automáticamente. (Por favor sé paciente, el proceso de actualización puede tardar un tiempo).
 
-### Account Software:  monero-wallet-gui
+### Software de cuenta: monero-wallet-gui
 
-Lauch `monero-wallet-gui`. If this is the first time you launched it go to the next step, otherwise click `Cancel`:
+Abre `monero-wallet-gui`. Si es la primera vez que lo abres ve al siguiente paso, de otro modo, haz clic en  `Cancel`:
 
 ![cancel opening](png/restore_account/cancel-opening.png)
 
-Select your appropriate language `Español`:
+Selecciona el lenguaje apropiado `Español`:
 
 ![choose language](png/restore_account/choose-language.png)
 
-Click on `Restore wallet from keys or mnemonic seed`:
+Haz clic en  `Restaurar monedero de claves o semilla mnemónica`:
 
 ![choose restore](png/restore_account/choose-restore.png)
 
-Keep `Restore from seed` selected, give your wallet a name & storage location, and complete the `Enter your 25 (or 24) word mnemonic seed`. Optionaly, specify a `Restore height (optional)` to avoid scanning of oldest blocks. Then click the `Right` arrow:
+Mantén `Restaurar por semilla` seleccionado, da un nombre a tu monedero y ubicación de guardado, y llena `Ingresa tu semilla mnemónica de 25 (o 24) palabras`. Opcionalmente, especifica la `Altura de restaurado (opcional)` para evitar escaneo de bloques viejos. Haz clic en la flecha de `Siguiente`:
```

Comment:
I will suggest changing in the [`monero-wallet-gui`](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts), some of the translations:

 - `Restore from seed` to `Restaurar por semilla`, it has [`Restaurar desde semilla`](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts#L2330).

 - `Enter your 25 (or 24) word mnemonic seed` to `Ingresa tu semilla mnemónica de 25 (o 24) palabras`, it has [`Escribir las 25 (o 24) palabras de la semilla mnemotécnica`](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts#L2379)


I found these changes much better. :) :+1: 
I will also be updating the word `mnemónica`, it has [`Semilla mnemotécnica`](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts#L355)

But for the translation made for `Restore height (optional)` to `Altura de restaurado (opcional)`, the `monero-wallet-gui` has [`Restaurar altura (opcional)`](https://github.com/monero-project/monero-gui/blob/master/translations/monero-core_es.ts#L2360). This one sounds more precise. I will suggest to change it in this translation. 

The word **"específica"** is missing the tilde. 

I suggest the following:
"Opcionalmente, **específica** la altura en `Restaurar altura (opcional)` para evitar escaneo de bloques viejos."

Reply:
Ok guys about his comment. I think I missed several things I had already encountered in the past with some of these words and by mistake forgot to address them here.

"Semilla mnemónica" is a slang. The word mnemotécnica (mnemonic) comes from [Regla Mnemotécnica](https://es.wikipedia.org/wiki/Regla_mnemot%C3%A9cnica); [Mnemotecnia](https://www.mnemotecnia.es/inicio), so I suggest we shouldn't change it. I remember that I had "semilla mnemotécnica" in the terminology and now that I take a look it has been changed so I will add a comment about it.

I will be putting a strikethrough in some parts of the sentences but not deleting what I wrote, so we can keep up the history.

Reply:
I suggest that you don't use too much Wikipedia. About mnemonic, I've found two definitions in the RAE dictionary. One is about:

**mnemotecnia**
Del gr. μνήμη mnḗmē 'memoria' y -tecnia.
1. f. Procedimiento de asociación mental para facilitar el recuerdo de algo.

**mnemónico, ca**
1. adj. Perteneciente o relativo a la memoria.
2. f. mnemotecnia.

So it's the same, like a conjugation of the same word, so we can use _semilla mnemónica_. Yes, "mnemotecnia" is the art and the main word, but mnemónico is also a related-word for that.

We can make a decision and then change of left the word that way. You can also search for the definitions in Google.

Reply:
Thank you for the clarification and addressing this correclty.I will update what I changed and suggest the update in the `monero-wallet-gui`. :)

Reply:
There's some people and pages who uses "semilla mnemotécnica", but using some logic, it's just more OK to me to use "mnemónica" since you don't use, in example, something similar, I don't have something in mind but some techniques have another conjugation when you talk about a particular aspect of it, not sure if you follow :)

Reply:
Yes is ok, I really didn't know that "mnemónica" existed as a word. It really follows what you are pointing out in your last comment. I would keep "mnemónica" because it exists as a word, and it really makes sense. It even makes it shorter, and better understandable with the own English word "mnemonic". I think these discussions are good and productive, maybe they are a little overwhelming, but the discussions will help other contributors when they get around. These investigations give grammatical structure to the crypto-space where there isn't any type of grammatical investigation yet. I'm really thankful for your patience and keeping the hard work. :)

Reply:
Thanks to you too, it's just the way it should be :)